src/gulpfile.js

```diff
@@ -0,0 +1,107 @@
+/* jshint node:true */
+'use strict';
+// generated on 2015-01-19 using generator-gulp-webapp 0.2.0
+var gulp = require('gulp');
+var shell = require('gulp-shell');
+var $ = require('gulp-load-plugins')();
+
+gulp.task('collectstatic', shell.task(['python /vagrant/python/cac_tripplanner/manage.py collectstatic --noinput']));
```

Comment:
Not sure if there's a good way to template this out or not, but that absolute path in there probably isn't going to work for very long.